tasks/skylab/ConvertStarOutput.wdl

```diff
@@ -0,0 +1,54 @@
+version 1.0
+
+task ConvertStarOutput {
+
+  input {
+    File barcodes
+    File features
+    File matrix
+
+    #runtime values
+    String docker = " quay.io/kishorikonwar/secondary-analysis-python3-scientific:utils2"
```

Comment:
Just a note to update this to an official docker image before release.

Reply:
updated